lib/songkick/transport/curb.rb

```diff
@@ -63,6 +61,11 @@ def execute_request(req)
           connection.http(req.verb.upcase)
         end
 
+        if self.instrumenter
+          self.instrumentation_payload_extras[:connect_time] = connection.connect_time
```

Comment:
`Curl::Easy` also has its own notion of `total_time`, wasn't sure whether this should be put into the payload. Should I prefix these like `curl_connect_time`, `curl_name_lookup...` etc?